tests/test_kubernetes_tools.py

```diff
@@ -1613,6 +1613,7 @@ def test_get_pod_template_spec(
             "paasta.yelp.com/service": mock_get_service.return_value,
             "paasta.yelp.com/autoscaled": "false",
             "registrations.paasta.yelp.com/kurupt.fm": "true",
+            "yelp.com/owner": "compute_infra",
```

Comment:
We can probably make this `compute_infra_platform_experience` to be more specific (not sure on the formatting of that name if there's any abbreviations etc.

Reply:
ack, I am much more aligned to keep it as `compute_infrastructure_platform_experience`, because that would allow us to use the ownership API - https://ownership.yelpcorp.com/v3/owners?limit=10&offset=0&owner_name=compute%20infrastructure%20platform%20experience 

for ee-metrics, any hard opinions? @mattmb

Reply:
Discussed with mmb to use `compute_infrastructure_platform_experience` discussion - https://yelp.slack.com/archives/D02V9TE4DRR/p1663670316846139